app/frontend/src/cljs/teet/meeting/meeting_view.cljs

```diff
@@ -966,18 +967,19 @@
        [:div {:class (<class common-styles/heading-and-action-style)}
         [typography/Heading2 {:class (<class common-styles/flex-align-center)}
          title (when number (str " #" number)) (when locked? [icons/action-lock])]
+        [:div {:class (<class common-styles/flex-align-end)}
```

Comment:
formatting issues here. The authorization-context component should be aligned to the right since it's inside this div. Try to use paredit to format this.